### PR TITLE
Condense Mantle of the Frozen Heart spell effect

### PR DESCRIPTION
**Pull request by ammalagonc**
I got 3 of the 4 effects down. However, I could not get the damage note on Heart of Ice to work. I opened an issue—once it is resolved, I'll fold it into this.

### CODE REVIEW

File: packs/spell-effects/spell-effect-mantle-of-the-frozen-heart.json [hunk 1]
```diff
@@ -0,0 +1,193 @@
...
+                        "value": "heart-of-ice"
+                    },
+                    {
+                        "label": "Ice Glide",
+                        "value": "ice-glide"
+                    },
+                    {
+                        "label": "Icy Claws",
+                        "value": "icy-claws"
+                    }
```
Comment (pedrogrullada) [inline packs/spell-effects/spell-effect-mantle-of-the-frozen-heart.json:46]: You're missing localization for these

File: packs/spell-effects/spell-effect-mantle-of-the-frozen-heart.json [hunk 2]
```diff
@@ -0,0 +1,193 @@
...
+            {
+                "key": "FlatModifier",
+                "predicate": [
+                    {
+                        "or": [
+                            "frozen-heart-first:ice-glide",
+                            "frozen-heart-second:ice-glide"
+                        ]
+                    }
+                ],
```
Comment (pedrogrullada) [inline packs/spell-effects/spell-effect-mantle-of-the-frozen-heart.json:118]: This speed bonus is conditional, so you'll need a toggleable roll option for it, which you can have disabled if the `ice-glide` roll options are not active.

File: packs/spell-effects/spell-effect-mantle-of-the-frozen-heart.json [hunk 3]
```diff
@@ -0,0 +1,193 @@
...
+            {
+                "key": "BaseSpeed",
+                "predicate": [
+                    {
+                        "or": [
+                            "frozen-heart-first:icy-claws",
+                            "frozen-heart-second:icy-claws"
+                        ]
+                    }
+                ],
```
Comment (pedrogrullada) [inline packs/spell-effects/spell-effect-mantle-of-the-frozen-heart.json:176]:
```suggestion
                "predicate": [
                    "hands-free:2",
                    {
                        "or": [
                            "frozen-heart-first:icy-claws",
                            "frozen-heart-second:icy-claws"
                        ]
                    }
                ],
```

File: packs/spell-effects/spell-effect-mantle-of-the-frozen-heart.json (hunk 2 above)
Comment (ammalagonc) [reply packs/spell-effects/spell-effect-mantle-of-the-frozen-heart.json]: Added - check when you get a moment.

File: static/lang/re-en.json [hunk 4]
```diff
@@ -5153,6 +5153,21 @@
...
+                   "Label": {
+                       "ChillingIce": "Chilling Ice",
+                       "HeartOfIce": "Heart of Ice",
+                       "IceGlide": "Ice Glide",
+                       "IceGlideSurface": "Moving along a smooth surface",
+                       "IcyClaws": "Icy Claws"
+                   },
+                   "RollOptionLabel": {
+                    "First": "Mantle of the Frozen Heart - First Effect",
+                    "Second": "Second Effect"
```
Comment (pedrogrullada) [inline static/lang/re-en.json:5167]:
```suggestion
                    "First": "Mantle of the Frozen Heart — First ability",
                    "Second": "Mantle of the Frozen Heart — Second ability"
```

File: packs/spell-effects/spell-effect-mantle-of-the-frozen-heart.json [hunk 5]
```diff
@@ -0,0 +1,203 @@
...
+                "option": "ice-glide-active",
+                "predicate": [
+                    {
+                        "or": [
+                            "frozen-heart-first:ice-glide", 
+                            "frozen-heart-second:ice-glide"]
+                    }
+                ],
+                "toggleable": true
+            },
```
Comment (pedrogrullada) [inline packs/spell-effects/spell-effect-mantle-of-the-frozen-heart.json:109]:
```suggestion
                "domain": "all",
                "key": "RollOption",
                "label": "PF2E.SpecificRule.SpellEffectMantle.FrozenHeart.Label.IceGlideSurface",
                "option": "ice-glide-active",
                "disabledIf": [
                    {
                        "nor": [
                            "frozen-heart-first:ice-glide", 
                            "frozen-heart-second:ice-glide"
                        ]
                    }
                ],
                "toggleable": true
            },
```

**Review by pedrogrullada — APPROVED** [review]

**Pull request merged**